Kick Off#2479/README.md

```diff
@@ -0,0 +1,9 @@
+# 🖐 Hi there, I'm *JosofiCode*
+### ➖ I'm glad to see you _here_, let me tell you something about me...
+### ➖ I’m currently learning Python with Platzi and JavaScript with Guillermo Rodas with a Fullstack Bootcamp.
+### ➖ I'm a software developer very interested in Backend development
+
+### ➖ My skills are:
+⚫ Python
+⚫ SQL
+⚫ JavaScript
```

Comment:
If you can add the terminal commands that you mostly use, it would be nice. That is part of the exercise of the first class.

Reply:
Thanks you! I did it.